src/include/cell_inline.h

```diff
@@ -1311,20 +1311,19 @@ __wt_cell_unpack_delta_int(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *page_
   const WT_DELTA_HEADER *dsk, WT_DELTA_CELL_INT *cell, WT_CELL_UNPACK_DELTA_INT *unpack_delta)
 {
     WT_DECL_RET;
-    uint8_t flags;
     const uint8_t *p;
 
     WT_UNUSED(dsk);
 
-    flags = cell->__chunk[0];
+    unpack_delta->flags = cell->__chunk[0];
     p = (uint8_t *)&cell->__chunk[1];
 
     /* Unpack the key. */
     __wt_cell_unpack_kv(session, page_dsk, (WT_CELL *)p, &unpack_delta->key);
     p += unpack_delta->key.__len;
 
     /* Optionally unpack the value if it exists. */
-    if (!LF_ISSET(WT_DELTA_IS_DELETE)) {
+    if (!F_ISSET(unpack_delta, WT_DELTA_IS_DELETE)) {
```

Comment:
Should be WT_DELTA_INT_IS_DELETE?

Reply:
great catch, thanks!!

Reply:
Updated in [b2c1c69](https://github.com/wiredtiger/wiredtiger/pull/11627/commits/b2c1c69cd3a8d110fff8c439ddf6dbc11f7b62de)